setup.py

```diff
@@ -0,0 +1,81 @@
+import codecs
+
+from setuptools import setup, find_packages
+
+with codecs.open("README.md", encoding="utf-8") as f:
+    readme = f.read()
+
+setup(
+    name="cloudimized",
+    version="1.0.0",
+    description='Google Cloud Platform (GCP) configuration scanning tool',
+    long_description=readme,
+    long_description_content_type="text/markdown",
+    url="https://github.com/egnyte/cloudimized",
+    author="Egnyte and Contributors",
+    classifiers=[
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+        "Development Status :: 5 - Production/Stable",
+        "Intended Audience :: Information Technology",
+        "Intended Audience :: System Administrators",
+        "License :: OSI Approved :: MIT License",
+        "Operating System :: POSIX :: Linux",
+        "Operating System :: MacOS",
+    ],
+    packages=find_packages(),
+    package_data={"": ["LICENSE", "*.md", "config-example.yml"]},
+    include_package_data=True,
+    install_requires=[
```

Comment:
With recent new feature requirements list needs to be updated.

Reply:
Updated.